casr/tests/tests.rs

```diff
@@ -4562,3 +4582,597 @@ fn test_casr_cluster_d_python() {
 
     let _ = std::fs::remove_dir_all(&paths[1]);
 }
+
+#[test]
+#[cfg(target_arch = "x86_64")]
```

Comment:
What about other archs?

Reply:
As I see from other tests, the main concept is to avoid running tests on other archs. Is this incorrect?

Reply:
If a test failed you need remove all files manually that's not convenient

Reply:
I so know, we try to support all archs

Reply:
We tend to support all archs but not to run all the tests on them due to huge execution time.